Sources/WasmKit/Translator.swift

```diff
@@ -2257,6 +2257,16 @@ struct InstructionTranslator<Context: TranslatorContext>: InstructionVisitor {
             return .tableSize(Instruction.TableSizeOperand(tableIndex: table, result: LVReg(result)))
         }
     }
+
+    mutating func visitUnknown(_ opcode: [UInt8]) throws -> Bool {
+        guard opcode.count == 1 && opcode[0] == 0xFF else {
```

Comment:
Could you gate this behind a Translator flag so that it won't accidentally interpret unrecognized opcode when the debugging mode is not enabled for the function?

Reply:
Added `isDebugging` property and initializer parameter here, which is currently set to `false`.